hudi-client/hudi-client-common/src/main/java/org/apache/hudi/callback/client/http/HoodieWriteCommitHttpCallbackClient.java

```diff
@@ -43,36 +48,42 @@ public class HoodieWriteCommitHttpCallbackClient implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(HoodieWriteCommitHttpCallbackClient.class);
 
   public static final String HEADER_KEY_API_KEY = "HUDI-CALLBACK-KEY";
+  public static final String HEADERS_DELIMITER = ";";
+  public static final String HEADERS_KV_DELIMITER = ":";
```

Comment:
Can these be private or package-protected?

Reply:
done